### PR TITLE
Document the storeFormat option for create database

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1577.surge.sh

Updated pages:
- [Store formats](https://neo4j-docs-operations-1577.surge.sh/operations-manual/5/database-internals/store-formats)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1577.surge.sh

Updated pages:
- [Create, start, and stop databases](https://neo4j-docs-operations-1577.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Store formats](https://neo4j-docs-operations-1577.surge.sh/operations-manual/5/database-internals/store-formats)

File: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc [hunk 1]
```diff
@@ -104,7 +104,7 @@ The `CREATE DATABASE` command can have a map of options, e.g. `OPTIONS {key: 'va
 | `use`
 |
 Controls how the system handles existing data on disk when creating the database.
-Currently this is only supported with `existingDataSeedInstance` and must be set to `use` which indicates the existing data files should be used for the new database.
+Currently, this is only supported with `existingDataSeedInstance` and must be set to `use` which indicates the existing data files should be used for the new database.
```
Comment (mnd999) [inline modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc:107]: There is also `seedURI` now isn't there? I'm not sure if that requires `existingData` or not though. I think it probably should.

File: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc [hunk 2]
```diff
@@ -130,9 +130,14 @@ Defines credentials that need to be passed into certain seed providers.
...
 |
-Defines level of enrichment applied to transaction logs for Change Data Capture (CDC) purposes.
+Defines the level of enrichment applied to transaction logs for Change Data Capture (CDC) purposes.
 
 For details about enrichment mode, see link:{neo4j-docs-base-uri}/cdc/current/get-started/self-managed/#set-enrichment-mode/[Change Data Capture Manual -> Enable CDC on self-managed instances -> Set the enrichment mode].
+
+| `storeFormat`
+| `aligned` \| `standard` \| `high_limit` \| `block`
+|
+Defines the store format of a new database.
```
Comment (mnd999) [inline modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc:140]: We should also state that this is only effective if the database is created new. If the store is seeded with `seedURI`, or `existingDataSeedInstance` or if the command is being used to mount existing store files already on the disk they will use their current store format - there will not be any migration.

Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc:140]:
```suggestion
Defines the store format if the database created is new.
If the store is seeded with `seedURI` or `existingDataSeedInstance`, or if the command is being used to mount pre-existing store files already present on the disk they will use their current store format without any alterations.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1577.surge.sh

Updated pages:
- [Create, start, and stop databases](https://neo4j-docs-operations-1577.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Store formats](https://neo4j-docs-operations-1577.surge.sh/operations-manual/5/database-internals/store-formats)

File: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc (hunk 1 above)
Comment (renetapopova) [inline modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc:107]:
```suggestion
Currently, this is only supported with `existingDataSeedInstance` and `seedURI`, and must be set to `use` which indicates the existing data files should be used for the new database.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1577.surge.sh

Updated pages:
- [Create, start, and stop databases](https://neo4j-docs-operations-1577.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Store formats](https://neo4j-docs-operations-1577.surge.sh/operations-manual/5/database-internals/store-formats)

File: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc [hunk 3]
```diff
@@ -34,11 +34,11 @@ CREATE DATABASE customers
 ----
 
 In Neo4j, the default store format for all new databases is `aligned`.
-If you want to change it, you can set a new value for the xref:configuration/configuration-settings.adoc#config_db.format[`db.format`] configuration in the _neo4j.conf_ file.
-
+If you want to change it, you can set a new value for the xref:configuration/configuration-settings.adoc#config_db.format[`db.format`] configuration in the _neo4j.conf_ file. +
+Alternatively, you can set the store format of a new database using the `CREATE DATABASE databasename OPTIONS {storeFormat: 'the-new-format'}` command. +
```
Comment (renetapopova) [inline modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc:38]:
```suggestion
Alternatively, you can set the store format of new databases using the `CREATE DATABASE databasename OPTIONS {storeFormat: 'the-new-format'}` command. 
However, if the store is seeded with `seedURI` or `existingDataSeedInstance`, or if the command is being used to mount pre-existing store files already present on the disk, they will use their current store format without any alterations.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1577.surge.sh

Updated pages:
- [Create, start, and stop databases](https://neo4j-docs-operations-1577.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Store formats](https://neo4j-docs-operations-1577.surge.sh/operations-manual/5/database-internals/store-formats)

**Review by emmaholmbergohlsson — APPROVED** [review]
It's fine from kernel perspective

**Review by NataliaIvakina — APPROVED** [review]
I think it's good from the docs perspective

**Pull request merged**